src/pages/group/index.tsx

```diff
@@ -13,15 +14,28 @@ export default function GroupsSearch() {
         <meta name="description" content="App description" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <div className="flex flex-col gap-4 w-full">
+      <div className="flex w-full flex-col gap-4">
+        <CreateGroupModal />
         {groupsList.data?.groups.map((group) => (
-          <div key={group.id} className="flex ring-base-500 rounded-lg bg-base-100 ring-1 p-3">
-            <Link href={`/${group.id}`} className="relative h-10 w-10 overflow-clip rounded-lg bg-secondary ring-1 ring-base-200">
-              <Image src={`${group.photoUrl}`} alt={`Image for ${group.name}`} width={50} height={50} unoptimized={true}/>
+          <div
+            key={group.id}
+            className="flex rounded-lg bg-base-100 p-3 ring-1 ring-base-500"
+          >
+            <Link
+              href={`/${group.id}`}
```

Comment:
Is this linking where intended? Not sure if we're wanting to point to `/group/${group.id}` or `/${group.id}`.

Reply:
@redconOne I believe this is already being addressed in Derek's PR he submitted last night.  Not sure how that works if I change it after his PR is approved?

Reply:
should be `/group/${group.id}`

Reply:
Updated.